spec/spec_helper_acceptance.rb

```diff
@@ -14,7 +14,7 @@
     end
     # Install git, it's a dependency for inspec profiles
     # Found this when experiencing https://github.com/chef/inspec/issues/1270
-    install_package(host, 'git')
+    # install_package(host, 'git')
```

Comment:
? if you don't need this remove the line and its comments?

Reply:
I commented this out because to get around it with the idea that I would look at compliance later and forgot.  Seems like compliance does not need it.  I am removing it becuase trying to install a package before setting up the redhat repos causes the tests to fail for redhat.